`svelte-migrate package` is broken

### Describe the bug

The `utils.js` file is missing in the [1.1.0 release](https://www.npmjs.com/package/svelte-migrate/v/1.1.0?activeTab=explore).

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module './node_modules/svelte-migrate/utils.js' imported from ./node_modules/svelte-migrate/migrations/package/index.js
```


### Reproduction

Run `npx svelte-migrate package` as guided. https://github.com/sveltejs/kit/pull/8922

### Logs

```Shell
Error [ERR_MODULE_NOT_FOUND]: Cannot find module './node_modules/svelte-migrate/utils.js' imported from ./node_modules/svelte-migrate/migrations/package/index.js
    at new NodeError (node:internal/errors:387:5)
    at finalizeResolution (node:internal/modules/esm/resolve:330:11)
    at moduleResolve (node:internal/modules/esm/resolve:907:10)
    at defaultResolve (node:internal/modules/esm/resolve:1115:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:841:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```


### System Info

```Shell
System:
    OS: macOS 13.1
    CPU: (8) arm64 Apple M1
    Memory: 56.28 MB / 8.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.19.1 - ~/.nvm/versions/node/v16.19.1/bin/node
    npm: 8.19.3 - ~/.nvm/versions/node/v16.19.1/bin/npm
  Browsers:
    Chrome: 110.0.5481.100
    Firefox: 109.0.1
    Safari: 16.2
  npmPackages:
    @sveltejs/adapter-auto: ^2.0.0 => 2.0.0 
    @sveltejs/kit: ^1.5.0 => 1.7.1 
    @sveltejs/package: ^1.0.0 => 1.0.2 
    svelte: ^3.55.1 => 3.55.1 
    svelte-migrate: ^1.1.0 => 1.1.0 
    vite: ^4.0.0 => 4.1.1
```


### Severity

annoyance

### Additional Information

_No response_

--- a/.changeset/fast-mugs-cover.md
+++ b/.changeset/fast-mugs-cover.md
@@ -1,0 +1,5 @@
+---
+'svelte-migrate': patch
+---
+
+fix: include utils in migrate's published files

--- a/packages/migrate/package.json
+++ b/packages/migrate/package.json
@@ -18,7 +18,8 @@
 	"files": [
 		"bin.js",
 		"migrations",
-		"utils.js"
+		"utils.js",
+		"!migrations/**/*.spec.js"
 	],
 	"dependencies": {
 		"kleur": "^4.1.5",

--- a/packages/migrate/package.json
+++ b/packages/migrate/package.json
@@ -17,7 +17,8 @@
 	},
 	"files": [
 		"bin.js",
-		"migrations"
+		"migrations",
+		"utils.js"
 	],
 	"dependencies": {
 		"kleur": "^4.1.5",

--- a/packages/migrate/package.json
+++ b/packages/migrate/package.json
@@ -19,7 +19,8 @@
 		"bin.js",
 		"migrations",
 		"utils.js",
-		"!migrations/**/*.spec.js"
+		"!migrations/**/*.spec.js",
+		"!migrations/**/samples.md"
 	],
 	"dependencies": {
 		"kleur": "^4.1.5",

--- a/packages/migrate/utils.js
+++ b/packages/migrate/utils.js
@@ -1,5 +1,5 @@
-import fs from 'fs';
-import path from 'path';
+import fs from 'node:fs';
+import path from 'node:path';
 import colors from 'kleur';
 import ts from 'typescript';
 import MagicString from 'magic-string';

